chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/CertifiedProductDownloadableResourceCreatorJob.java

```diff
@@ -122,23 +122,17 @@ public void accept(final CertifiedProductSearchDetails cp) {
                 edition);
     }
 
-    private void initializeWritingToFiles() throws IOException {
-        xmlPresenter = new CertifiedProductXmlPresenter();
+    private void initializeWritingToFiles(CertifiedProductXmlPresenter xmlPresenter, 
+            CertifiedProductCsvPresenter csvPresenter) throws IOException {
```

Comment:
Should these be `final`? Checkstyle likely thinks so

Reply:
Change made